src/components/ShowItemList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import firebase from 'firebase/app';
+import 'firebase/firestore';
+import '../lib/firebase';
+
+function ShowItemList() {
+  const [value, loading, error] = useCollection(
+    firebase.firestore().collection('items'),
+  );
+
+  return (
+    <div>
+      {error && <p>Error</p>}
+      {loading && <p>Loading..</p>}
+      {value && (
```

Comment:
I love that you are using [short-circuit conditionals](https://codeburst.io/javascript-short-circuit-conditionals-bbc13ac3e9eb) here 🙌. It is a great way to shorten the evaluation of boolean values. Ternary operators would be another approach to achieve the same result. But I also prefer short-circuit approach.